miner/worker.go

```diff
@@ -652,7 +652,7 @@ func (self *worker) commitNewWork() {
 			log.Warn("Can't find coinbase account wallet", "coinbase", self.coinbase, "err", err)
 			return
 		}
-		if self.config.XDPoS != nil && self.chain.Config().IsTIPXDCX(header.Number) {
+		if self.config.XDPoS != nil && self.chain.Config().IsTIPXDCX(header.Number) && !self.chain.Config().IsTIPXDCXDisable(header.Number) {
```

Comment:
why not add the condition into the `IsTIPXDCX` directly? and rename the `IsTIPXDCX` to something like `isXDCxEnabled` etc?

Reply:
Rename to `IsTIPXDCXMiner`